src/main/java/duke/Tasks/Event.java

```diff
@@ -0,0 +1,36 @@
+package duke.Tasks;
+
+import duke.DateTimeFormat;
+import duke.Exceptions.DescriptionException;
+import java.time.LocalDateTime;
+
+/**
+ *
+ */
+public class Event extends Task {
+
+    protected LocalDateTime by;
+    protected LocalDateTime to;
+    protected String strBy;
+    protected String strTo;
+
+    public Event(String description, String by, String to) throws DescriptionException {
+        super(description);
+        this.strBy = by;
+        this.strTo = to;
```

Comment:
Could potentially try to separate by logical units.

Reply:
hihi sorry what do you mean by this?